src/systems/userinput/userinput.js

```diff
@@ -155,7 +235,9 @@ AFRAME.registerSystem("userinput", {
       e => {
         for (const device of this.activeDevices) {
           if (device.gamepad === e.gamepad) {
+            console.warn("NEED TO UPDATE REGISTERED MAPPINGS WHEN GAMEPAD DISCONNECTED!");
```

Comment:
Need to be able to figure out what should be in registeredMappings based on the active devices and whether or not you're in vr